src/services/urlService.retry.test.ts

```
@@ -0,0 +1,90 @@
+}));
+
+const MOCK_URL = "https://mock-retry.com";
+
+const OK_HTML = "<html><head><title>Home</title></head><body>ok</body></html>";
+
+const SOFT_404_HTML =
+  "<html><head><title>404 Not Found</title></head><body>oops</body></html>";
+
+describe("checkUrl retry logic", () => {
+  it("succeeds on the first attempt and sets attempts to 1", async () => {
+    server.use(http.get(MOCK_URL, () => HttpResponse.html(OK_HTML, { status: 200 })));
```

ESLint

[error] Replace `http.get(MOCK_URL,·()·=>·HttpResponse.html(OK_HTML,·{·status:·200·}))` with `⏎······http.get(MOCK_URL,·()·=>·HttpResponse.html(OK_HTML,·{·status:·200·})),⏎····`

(prettier/prettier)